lib/hooks/create/project.js

```diff
@@ -135,6 +135,16 @@ module.exports = settings => {
             return model.setStatus(awaitingEndorsement.id);
           });
 
+      case 'transfer-draft':
+        return Promise.resolve()
+          .then(() => Project.query().findById(id))
+          .then(project => {
+            if (project.status !== 'inactive') {
+              throw new BadRequestError(`cannot perform 'transfer-draft' action for a non-draft project`);
+            }
+            return model.setStatus(resolved.id);
```

Comment:
I think these are fine to autoresolve. I can't see a good reason to leave a task behind for a fairly straightforward edit to a draft PPL.

Reply:
Sorted.